src/main/java/core/basesyntax/model/ma/Coach.java

```diff
@@ -1,5 +1,10 @@
 package core.basesyntax.model.ma;
 
+import javax.persistence.Entity;
+import javax.persistence.PrimaryKeyJoinColumn;
+
+@Entity
+@PrimaryKeyJoinColumn(name = "coachId")
```

Comment:
can we go without this annotation?

Reply:
we can, but I decided that it is more comfortable)